studio-docs/source/explorer/sandbox.mdx

```diff
@@ -58,6 +58,8 @@ From [Sandbox](https://studio.apollographql.com/sandbox/), open the **Schema Dif
 
 <img class="screenshot" src="../img/sandbox-diff.jpg" alt="Schema diff in Sandbox" />
 
+> If you are introspecting a subgraph, you will see the subgraph sdl, _not_ the federation-generated subgraph schema. You can see the federation-generated subgraph schema on the SDL page in Sandbox.
```

Comment:
Hm, this is a bit confusing 🤔 . Maybe like:

"If you're using Sandbox with a subgraph, this diff shows the schema as originally defined, _not_ the subgraph schema that includes federation-specific fields like `Query._entities`. You can view the subgraph schema from the SDL page in Sandbox."

Is this factually accurate or have I lost some of the meaning?